qemu/tests/virtio_fs_share_data.py

```diff
@@ -136,6 +136,66 @@ def viofs_svc_stop_start(action, cmd, expect_status):
             test.fail("Could not %s VirtioFsSvc service, "
                       "detail: '%s'" % (action, output))
 
+    def enable_debug_log(session):
+        """
+        Only for windows guest, enable debug log in guest.
+        :param session: vm session
+        :return: session
+        """
+        viofs_debug_enable_cmd = params.get("viofs_debug_enable_cmd")
+        viofs_log_enable_cmd = params.get("viofs_log_enable_cmd")
+        if viofs_debug_enable_cmd and viofs_log_enable_cmd:
+            error_context.context("Check if virtiofs debug log is enabled in guest.", test.log.info)
+            cmd = params.get("viofs_reg_query_cmd")
+            ret = session.cmd_output(cmd)
+            if "debugflags" not in ret.lower() or "debuglogfile" not in ret.lower():
+                error_context.context("Configure virtiofs debug log.", test.log.info)
+                for reg_cmd in (viofs_debug_enable_cmd, viofs_log_enable_cmd):
+                    error_context.context("Set %s " % reg_cmd, test.log.info)
+                    s, o = session.cmd_status_output(reg_cmd)
+                    if s:
+                        test.fail("Fail command: %s. Output: %s" % (reg_cmd, o))
+                error_context.context("Reboot guest.", test.log.info)
+                session = vm.reboot()
+            else:
+                test.log.info("Virtiofs debug log is enabled.")
```

Comment:
Suggest to delete the `else` here, print the log directly.

Reply:
This is the previous code, I just moved the whole code, actually there is no change. And keep the log info is okay. What do you think?

Reply:
What I mean is that we can print the log in both if and else cases.

Reply:
The whole loginfo logic is:  "Check if virtiofs debug log is enabled in guest." if no,  "Configure virtiofs debug log.", if yes,  "Virtiofs debug log is enabled.".
As we talk in private, keep it.